angular_components/1-base/pubspec.yaml

```diff
@@ -19,3 +19,7 @@ transformers:
     resolved_identifiers:
       Random: 'dart:math'
 - dart_to_js_script_rewriter
+
```

Comment:
In Stagehand, we're still only using the commented-out dartdevc text. I think we should be consistent.